src/snowflake/snowpark/_internal/analyzer/analyzer_utils.py

```diff
@@ -737,13 +739,22 @@ def join_statement(
     )
 
 
+def get_comment_sql(comment: Optional[str]) -> str:
```

Comment:
nit: default to None?
```suggestion
def get_comment_sql(comment: Optional[str] = None) -> str:
```